VK_App/VK_App/Controller/FriendsViewController.swift

```diff
@@ -3,75 +3,187 @@
 
 import UIKit
 
+typealias Info = [Character: [(String, String)]]
 /// Страница с друзьями
```

Comment:
```suggestion
/// Экран с друзьями
```
Веб Страница ? Наташа ты в порядке?)